tools/azure-sdk-tools/packaging_tools/generate_client.py

```diff
@@ -2,11 +2,28 @@
 import logging
 from pathlib import Path
 from subprocess import run
-import sys
+import tomli
 
 _LOGGER = logging.getLogger(__name__)
 
 
+CONF_NAME = "pyproject.toml"
+
+def check_post_process(folder: Path) -> bool:
+    conf_path = folder / CONF_NAME
+    if not conf_path.exists():
+        return False
+
+    with open(conf_path, "rb") as fd:
+        return tomli.load(fd)["generate"]["autorest-post-process"]
```

Comment:
What's the best place to document this for folks to use?

Reply:
Adding a check here if there's not generate config in the toml

Reply:
> What's the best place to document this for folks to use?

I was going to suggest eng_sys_checks.md, but that's really focused on how to manage the CI checks. Maybe somewhere where we talk about generating python libraries? Here's the wiki for [swagger](https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-For-Swagger), not sure if we have one for typespec

Reply:
The default quick start describes how to generate with TypeSpec, but I think adding this in the swagger wiki makes the most sense. Thanks! :)